src/librustc_expand/build.rs

```diff
@@ -350,16 +350,10 @@ impl<'a> ExtCtxt<'a> {
     }
 
     pub fn expr_fail(&self, span: Span, msg: Symbol) -> P<ast::Expr> {
```

Comment:
Heh, this is still called "fail" (from before `fail!` was renamed to `panic!`).

Reply:
lol -- want to file a bug to followup or just noting it?